model-mesh/runtimes/kustomization.yaml

```diff
@@ -26,8 +26,8 @@ images:
     newTag: 0.5.2
 
   - name: ovms-1
-    newName: openvino/model_server
-    newTag: "2022.3"
+    newName: quay.io/opendatahub/openvino_model_server
+    newTag: "2022.3-release"
```

Comment:
in any non-master branch, digest: sha256xxxxxxxx format instead of newTag format should be used to support restricted network environment installs (on-prem) with mirrored repos and ImageContentSourcePolicy.

Reply:
this information has nothing to do with a disconnected environment.  The mirror command line is not referring this information.

Reply:
but the **ImageContentSourcePolicy generated by oc mirror**, or, in my case generated manually, needs sha256 type digests, not tags, in manifests. Note my content "restricted network environment installs (on-prem) with mirrored repos and ImageContentSourcePolicy"